Examples/Python/GroomUtils.py

```diff
@@ -304,16 +307,16 @@ def applyRigidAlignment(outDir, inDataListSeg, inDataListImg, refFile,
         if processRaw:
             cmd = ["shapeworks", 
                    "read-image", "--name", rawinname,
-                #    matrix created by icp is not used
+                #    matrix created by icp is not used so we use resize since icp resizes all images to the reference image which is a requirement for cropping (next step)
```

Comment:
Just to double check: we haven't modified icp in any way, correct? So it still does what it's supposed to do, just it's used.

Reply:
Yes